oleggerasimovich/parser.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import os
+
+from dotenv import load_dotenv
+import psycopg2
+
+load_dotenv()
+
+db = os.getenv('DATABASE')
+password = os.getenv('PASSWORD')
+host = os.getenv('HOST')
+user = os.getenv('USER')
+port = os.getenv('PORT')
+
+parser = argparse.ArgumentParser(description='Paste path to file')
+
+parser.add_argument('path', type=str, help='Put path to ur csv file')
+
+path = parser.parse_args()
```

Comment:
```suggestion
cli_arguments = parser.parse_args()
```

`parse_args()` returns the collection of arguments, not only the path.